modulefiles/module_base.wcoss2

```diff
@@ -0,0 +1,42 @@
+#%Module######################################################################
+##
+
+module load envvar/1.0
```

Comment:
this is loaded in `modulefiles/module-setup.sh.inc`
```suggestion
```

Reply:
Removed from both module-setup.sh.inc and module-setup.csh.inc at 0e9541da17b5d451d7fd1c89e68d7b077caa9678. Thanks!

Reply:
done, removed from module_base.wcoss2 and added back into module-setup.*.inc, thanks!